Feat: Add networks information in options (#98)
reown-com/reown-docs: docs/appkit/shared/options.mdx

createAppKit({ adapters, projectId, networks, ...options })
```

## networks

> You can set an array of networks that can be choosen from the library `@reown/appkit/networks`. This library get the list of viem networks and add the Solana ones.

Author: ```suggestion
You can set an array of networks that can be chosen from the library `@reown/appkit/networks`. This library get the list of viem networks and adds the Solana ones.
```

> You can set an array of networks that can be chosen from the library `@reown/appkit/networks`. This library get the list of viem networks and adds the Solana ones.

Reviewer: ```suggestion
You can set an array of networks that can be chosen from the `@reown/appkit/networks` library. This library retrieves the list of EVM networks from Viem and also includes the Solana networks.
```

Updating the text for correct grammar and mention that Viem only exposes EVM compatible networks.

```ts
import { mainnet, solana } from '@reown/appkit/networks'

createAppKit({
  // ...
  networks: [mainnet, solana]
})
```

> For custom networks, you can check the [specific documentation](/appkit/react/core/custom-networks).

Reviewer: ```suggestion
For custom networks, refer to this [doc page](/appkit/react/core/custom-networks).
```
